src/routes/Routes.tsx

```diff
@@ -69,6 +72,11 @@ import AdvancedPage from '~frame/Extensions/pages/AdvancedPage';
 import PermissionsPage from '~frame/Extensions/pages/PermissionsPage';
 import LazyConsensusPage from '~frame/Extensions/pages/LazyConsensusPage';
 import { ExtensionsContextProvider } from '~context/ExtensionsContext';
+import MembersPage from '~frame/v5/pages/MembersPage/MembersPage';
+import ContributorsPage from '~frame/v5/pages/ContributorsPage/ContributorsPage';
```

Comment:
`import ColonyDetailsPage from '~frame/Extensions/pages/ColonyDetailsPage';`

Please, change in other places and check your settings in VSCode to import the component

Reply:
Fixed